### PR TITLE
Support personal mode of RPC [WIP]

### LINKED ISSUE #198: UI: support personal account mode of RPC
Have a toggle to switch between `web3.eth` and `web3.personal` for accounts.

### PR DESCRIPTION
**Pull request by axic**
Fixes #198.

I've did this a while back, but I won't have time to finish with a nice UI.

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -646,6 +646,8 @@ var run = function () {
     startdebugging(txResult.transactionHash)
   })
 
+  $('#createAccount').click(function () { udapp.newAccount() })
```
Comment (axic) [inline src/app.js]: The UI must be refreshed (to populate the account dropdown) after a new account is created.

File: src/universal-dapp.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ function UniversalDApp (executionContext, options) {
 
   self.options = options || {}
   self.$el = $('<div class="udapp" />')
-  self.personalMode = self.options.personalMode || false
+  self.personalMode = true //self.options.personalMode || false
```
Comment (axic) [inline src/universal-dapp.js]: This should be changed based on the UI setting.

**Comment by chriseth** [comment]
Please fix tests and rebase.

File: src/app.js [hunk 3]
```diff
@@ -714,6 +715,8 @@ var run = function () {
     })
   })
 
+  $('#createAccount').click(function () { udapp.newAccount() })
```
Comment (axic) [inline src/app.js]: The UI must be refreshed (to populate the account dropdown) after a new account is created.

Comment (axic) [inline src/app.js]: The UI must be refreshed (to populate the account dropdown) after a new account is created.

File: src/app.js [hunk 4]
```diff
@@ -644,7 +644,8 @@ var run = function () {
   // ----------------- UniversalDApp -----------------
   var udapp = new UniversalDApp(executionContext, {
     removable: false,
-    removable_instances: true
+    removable_instances: true,
+    personalMode: $('#personalMode').val() === 'true'
```
Comment (axic) [inline src/app.js]: This should be changed as the UI is changed.

**Comment by axic** [comment]
> Please fix tests and rebase.

I won't have any time to finish this. See the above comments on what needs to be implemented.

**Comment by axic** [comment]
@yann300 can you look into this? Using the personal API would be very useful.

**Comment by yann300** [comment]
moved to https://github.com/ethereum/browser-solidity/pull/993

**Pull request closed without merging**